html/changelogs/wickecybs_medical.yml

```diff
@@ -0,0 +1,8 @@
+author: WickedCybs
+
+delete-after: True
+
+changes:
+  - rscadd: "Mention the pouches."
```

Comment:
I think you forgot to update this.

Reply:
PR is marked WIP.

Reply:
I'll be putting in more details once ironed out certain bits of the PR and have it ready to go.